src/clj/game/cards/resources.clj

```diff
@@ -146,11 +146,15 @@
                  :choices {:req #(and (facedown? %)
                                       (installed? %)
                                       (= "Runner" (:side %)))}
-                 :effect (req (let [temp (move state side target :play-area)
-                                    moved (move state side temp (type->rig-zone (:type temp)))]
-                                (card-init state side (make-eid state) moved {:resolve-effect false
-                                                                              :init-data false})))
-                 :msg (msg (str "turn" (:title target) "faceup"))}]}
+                 :effect (req (if (or (is-type? target "Event")
+                                      (has-subtype? target "Console"))
```

Comment:
Consoles are only trashed if another Console is installed
Almost correct 😜

Reply:
Bah!